### ISSUE TITLE
Adding new features

### ISSUE THREAD

**Issue by yogat3ch** [erikor/timeR#1]
Hi @erikor,
Thanks for putting together this package!
I've been using it to fetch some stuff from the Clockify API and found the need for a couple more functions. 
You can see the progress over at [yogat3ch/timeR](https://github.com/yogat3ch/timeR).
I've added `clockify_get_clients` and `clockify_get_reports` and reworked some of the internal functions for more robustness. 
The code adds a couple of dependencies. If you don't mind new dependencies and stylistically different code I'd be happy to create a PR.
Just let me know if you're interested!

**Comment by erikor** [comment]
Yes feel free to submit pull request. Also, I am not really developing this any more so let me know if you would like to take over the repo as maintainer.

**Comment by yogat3ch** [comment]
Hi @erikor,
Thanks for responding!
I'm using Clockify to report and do invoices with R so there's probably improvements I'll make over time. 
I'm happy to step in as maintainer if you're not into it anymore.
Is it on CRAN?
I'll take a look through the code and see if I have any questions about maintaining it.

**Comment by erikor** [comment]
Great. Can I transfer ownership of the repo to you?

**Comment by yogat3ch** [comment]
I have an open branch on my end, I've never transferred a repo when there were differences between the two. Do you know if the branch will be affected by the transfer? Should I submit it as a PR before transferring?
